### PR TITLE
Updated @gridsome/source-drupal README for 0.6.0 changes

### PR DESCRIPTION
**Pull request by matt-e-king**

### CODE REVIEW

**Comment by hjvedvik** [comment]
Thanks @matt-e-king Would it be better to use `:title` or `:field_name` in the route examples?

**Comment by matt-e-king** [comment]
@hjvedvik for `'node--article': '/articles/:slug',` I was still in the mind set that this plugin would be responsible for "slugifying" the `title` and adding a `slug` field to each node, worrying that if the route used `title` it would look something like `/article/This Is The Title`.

I now see upon closer inspection, you are already accounting for that :)

For `'taxonomy_term--tags': '/tags/:name'`, I know we are account for both `title` and `name` in the plugin, like so: `title: fields.title || fields.name` but my thinking was Drupal doesn't return a `title` for tags, it just returns `name` by default - so I was trying to keep that relationship clear. (Also, I don't see `field_name` in the node)

That said, I appreciate the consistency you are trying to create across ALL the source plugins, so I will definitely defer to your judgement. I've updated all example routes to `:title` in the latest commit. I've also updated the starter package to reflect that change.

**Comment by hjvedvik** [comment]
You're right :) I forgot that Drupal taxonomies just has `name`.

File: packages/source-drupal/README.md [hunk 1]
```diff
@@ -107,8 +107,8 @@ module.exports = {
       options: {
         baseUrl: 'https://somedrupalsite.pantheonsite.io',
         routes: {
-          'node--article': '/articles/:slug',
-          'taxonomy_term--tags': '/tags/:slug'
+          'node--article': '/articles/:title',
+          'taxonomy_term--tags': '/tags/:title'
```
Comment (hjvedvik) [inline packages/source-drupal/README.md]:
```suggestion
          'taxonomy_term--tags': '/tags/:name'
```

File: packages/source-drupal/README.md [hunk 2]
```diff
@@ -152,8 +152,8 @@ module.exports = {
         baseUrl: 'https://somedrupalsite.pantheon.io',
         exclude: [ ...defaultExcludes, 'user--user' ], // include the defaults
         routes: {
-          'node--article': '/articles/:slug',
-          'taxonomy_term--tags': '/tags/:slug'
+          'node--article': '/articles/:title',
+          'taxonomy_term--tags': '/tags/:title'
```
Comment (hjvedvik) [inline packages/source-drupal/README.md]:
```suggestion
          'taxonomy_term--tags': '/tags/:name'
```

**Comment by matt-e-king** [comment]
👌 Start package updated as well.

**Comment by hjvedvik** [comment]
Thank you @matt-e-king! It looks good now.

**Pull request merged**